catalog/app/website/pages/AwsMarketplace/Form.tsx

```diff
@@ -0,0 +1,347 @@
+import cx from 'classnames'
+import * as FF from 'final-form'
+import * as React from 'react'
+import * as RF from 'react-final-form'
+import * as M from '@material-ui/core'
+import { fade } from '@material-ui/core/styles'
+
+import TextField, { TextFieldProps } from 'components/Form/TextField'
+import log from 'utils/Logging'
+import * as validators from 'utils/validators'
```

Comment:
What are these validators doing? I was able to submit an email with no @ symbol. Not a top priority but worth consideration.

Reply:
There are basic validators here, and the only one I use is "required" (requires user to fill input field)

For email validation there is another check https://github.com/quiltdata/quilt/pull/3251/files#diff-635522e4007d740d663650285e405e29132f1801c2b070667c213915981f51cfR179

It's quite simple, just checks `@` and `.`, and I was unable to submit form email without a `@`

Reply:
OK. Am I correct in understanding that this is now applied? (I'm still a relative PR noob, so I can't tell whether this is in the latest rev)

Reply:
It was applied on 13 of January
I don't know, maybe you checked out the first revision only?